app/reports.py

```diff
@@ -71,9 +71,11 @@ def get_quality(source):
         breakpoint_index = CONCENTRATIONS.index(concentration)
         breakpoint = BREAKPOINTS[breakpoint_index]
 
-        index = (
-            (breakpoint[1] - breakpoint[0]) / (concentration[1] - concentration[0])
-        ) * (source.pm2dot5_average - concentration[0]) + breakpoint[0]
+        index = int(
```

Comment:
curious why was this needed? It isn't an actual array index.

Reply:
The casting to int is because int is expected.